### DISCUSSION TITLE
What shells are supported by sheldon?

### DISCUSSION THREAD

**Discussion by mothsarecool524** [rossmacarthur/sheldon#181] [Q&A]
E.g. `bash`, `zsh`, `fish` being the main three that most people use.
I know that `fish` has its own built-in plugin manager, `fisher`, but I can't help but wonder. :p 

Putting this list towards the top of the README, or somewhere in the Table of Contents, would be nice for those looking to use `sheldon` for the first time.

**Comment by rossmacarthur** [comment]
Currently only `bash` and `zsh` have good support and `fish` is experimental.

In theory someone could use sheldon with any shell if you override the correct global options: https://sheldon.cli.rs/Configuration.html#global-options

I will add a task to add this to the documentation! https://github.com/rossmacarthur/sheldon/issues/183

**Reply by xiaoxiangmoe** [reply]
@rossmacarthur Is there any way to make both zsh, bash and fish sheldon config works for single user?
